doc/man3/OpenSSL_version.pod

```diff
@@ -102,46 +102,47 @@ The value of B<OPENSSL_FULL_VERSION_STR>
 =item OPENSSL_CFLAGS
 
 The compiler flags set for the compilation process in the form
-"compiler: ..."  if available or "compiler: information not available"
+"compiler: ..."  if available, or "compiler: information not available"
 otherwise.
 
 =item OPENSSL_BUILT_ON
 
-The date of the build process in the form "built on: ..." if available
-or "built on: date not available" otherwise.
+The date of the build process in the form C<built on: ...> if available
+or C<built on: date not available> otherwise.
+The date would not be available in a  reproducible build, for example.
 
 =item OPENSSL_PLATFORM
 
-The "Configure" target of the library build in the form "platform: ..."
-if available or "platform: information not available" otherwise.
+The "Configure" target of the library build in the form C<platform: ...>
+if available, or "platform: information not available" otherwise.
 
 =item OPENSSL_DIR
 
-The "OPENSSLDIR" setting of the library build in the form "OPENSSLDIR: "...""
-if available or "OPENSSLDIR: N/A" otherwise.
+The "OPENSSLDIR" setting of the library build in the form C<OPENSSLDIR: "...">
+if available, or "OPENSSLDIR: N/A" otherwise.
 
 =item OPENSSL_ENGINES_DIR
 
-The "ENGINESDIR" setting of the library build in the form "ENGINESDIR: "...""
-if available or "ENGINESDIR: N/A" otherwise.
+The "ENGINESDIR" setting of the library build in the form C<ENGINESDIR: "...">
+if available, or "ENGINESDIR: N/A" otherwise.
 
 =item OPENSSL_MODULES_DIR
 
-The "MODULESDIR" setting of the library build in the form "MODULESDIR: "...""
-if available or "MODULESDIR: N/A" otherwise.
+The "MODULESDIR" setting of the library build in the form C<MODULESDIR: "...">
+if available, or C<MODULESDIR: N/A> otherwise.
 
 =item OPENSSL_CPU_INFO
 
 The current OpenSSL cpu settings.
 This is the current setting of the cpu capability flags. It is usually
 automatically configured but may be set via an environment variable.
 The value has the same syntax as the environment variable.
-For x86 the string looks like "CPUINFO: OPENSSL_ia32cap=0x123:0x456".
-Or "CPUINFO: N/A" if not available, e.g. no-asm build.
+For x86 the string looks like C<CPUINFO: OPENSSL_ia32cap=0x123:0x456>.
+Or C<CPUINFO: N/A> if not available, such as when configured with B<no-asm>.
 
 =back
 
-For an unknown B<t>, the text "not available" is returned.
+For an unknown B<t>, the text C<not available> is returned.
```

Comment:
`I<t>`

Reply:
are you sure about that?  it's a parameter

Reply:
Yes, parameters should be in italics. https://linux.die.net/man/7/man-pages

Reply:
Okay, thanks for the education. There were actually four places to fix.  Fixup commit pushed.